conf/routes

```diff
@@ -165,6 +165,8 @@ POST   /v3/scheduleplans           @org.sagebionetworks.bridge.play.controllers.
 GET    /v3/scheduleplans/:guid     @org.sagebionetworks.bridge.play.controllers.SchedulePlanController.getSchedulePlan(guid: String)
 POST   /v3/scheduleplans/:guid     @org.sagebionetworks.bridge.play.controllers.SchedulePlanController.updateSchedulePlan(guid: String)
 DELETE /v3/scheduleplans/:guid     @org.sagebionetworks.bridge.play.controllers.SchedulePlanController.deleteSchedulePlan(guid: String)
+POST   /v3/scheduleplans/events    @org.sagebionetworks.bridge.play.controllers.ActivityEventController.createActivityEvent(eventKey: String?= null, timestamp: String?=null)
```

Comment:
Doesn't this need to go before POST /v3/scheduleplans/:guid ? 

I think /v3/scheduleplans/events will call that method, with a guid of "events"